include/cvc5/cvc5_parser.h

```diff
@@ -77,6 +77,24 @@ class CVC5_EXPORT SymbolManager
    */
   const std::string& getLogic() const;
 
+  /**
+   * Get the list of sorts that have been declared via `declare-sort` commands.
+   * These are the sorts that are printed as part of a response to a
+   * `get-model` command.
+   *
+   * @return The declared sorts.
+   */
+  std::vector<Sort> getModelDeclaredSorts() const;
```

Comment:
Would it make sense to use `getDeclaredSorts()` and `getDeclaredTerms()` here? Users can query this independently from a model or the status of the input.

Reply:
Sure, this is probably a better name.